### PR TITLE
Change Plasma to Phoron

### PR DESCRIPTION
**Pull request by tigercat2000**
This commit changes every instance of 'Plasma' in text to 'Phoron'.

I've done this for consistency with Bay12, as we are still a downstream, after all.

Reasons for Phoron:
 - Plasma is actually a 4th state of matter, and is therefore not suitable to be the name of our fictional super-flammable material, and we can refer to said 4th state of matter ingame if there is not a magical gas that is named the same.
 - Increased consistency with Baystation12, therefore increasing upstream compatibility, which is hardly ever a bad thing.
	- This means it will be much easier to port more features from bay.
 - Orange looks a lot better than that odd shade of purple.
 - Much easier to pronounce. Foe-Ron!

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
About damn time. Dunno why this wasn't done earlier--this should have been done a long time ago. :+1: 

Oh, do you have plans on converting the gas icon to orange? Better readability and consistency.

**Comment by tigercat2000** [comment]
@Fox-McCloud Ah, I knew I was forgetting something, thanks!

**Comment by fj45** [comment]
I am going to assume this is just a sick joke.

**Comment by Fox-McCloud** [comment]
@fj45 

This was discussed quite extensively in the coderchat.

**Comment by fj45** [comment]
Well then......

This change shall happen over my dead, slightly charred body.

-This is something that can be forever debated, but remember, the general public is stupid in regards to science. Maybe it went something like: Fancy new element that burns like anything, media gets their mits on it...calls it "plasma" because Joe read a science textbook twenty years ago and that makes him know things, general public calls new element "plasma", scientists rip out their hair, new element mostly gets refered to as plasma.

Ect ect there are hundreds of ways you can go back and fourth on this. Plasma is a better name to me.

-The exact same thing could be said about TG. Don't we get a few more things from TG then bay anyway?

-Purple contrasts against fire far better. Never had an issue with the purple.

-I'm still holding out some slim hope this is a sick joke after this comment.

**Comment by Spacemanspark** [comment]
-Opens up github website.
-Oh hey look at all these notificat-
-PLASMA INTO PHORON!
-See's Fj45's icon among others.
-NOPE NOPE NOPE.

**Comment by ZomgPonies** [comment]
:laughing:

**Comment by fj45** [comment]
...ha...haha....ha...............................ha..................?.....

**Comment by Earthdivine** [comment]
:+1: from me, plasma being a catch all name or something that's a solid, gas, and liquid is silly. Make up a whole new name if your gunna start going to "magic elements".

On the other hand, now all we need is Ice Phoron and a Solid-Gas state and we have all of the types on matter!

**Comment by KasparoVy** [comment]
MAGE MATTER FOR THAT MAGICAL ELEMENT

:+1: woo phoron

**Comment by BiberDark** [comment]
Change the name keep the color.

**Comment by fj45** [comment]
Everything about this game is silly. Barely anything makes a good deal of sense.

**Comment by Mijhead** [comment]
:-1:

**Comment by KasparoVy** [comment]
desc = "When is a phoron giant not a plasma giant? When it's a bar serving plasma from a plasma giant." on line 349 of code/game/objects/structures/barsign.dm
should be "When is a phoron giant not a phoron giant? When it's a bar serving phoron from a phoron giant."

**Comment by PJB3005** [comment]
:+1:

**Comment by TheDZD** [comment]
:laughing:

**Comment by MactoPerFuror** [comment]
A long time ago, I might have been a bit fighty about this but, truly the "Plasma is actually a 4th state of matter, and is therefore not suitable to be the name of our fictional super-flammable material, and we can refer to said 4th state of matter ingame if there is not a magical gas that is named the same." argument has been bothering me for a long time now. So i gladly support this.

**Comment by PJB3005** [comment]
Oh, isn't plasma also that thing in a human's blood that carries everything that's no oxygen/white blood cells?

or does that only hold true in dutch

File: code/defines/vox_sounds.dm [hunk 1]
```diff
@@ -523,7 +523,7 @@ var/list/vox_sounds = list("," = 'sound/vox_fem/,.ogg',
 "personnel" = 'sound/vox_fem/personnel.ogg',
 "pipe" = 'sound/vox_fem/pipe.ogg',
 "plant" = 'sound/vox_fem/plant.ogg',
-"plasma" = 'sound/vox_fem/plasma.ogg',
+"phoron" = 'sound/vox_fem/plasma.ogg',
```
Comment (PJB3005) [inline code/defines/vox_sounds.dm]: Er, these vox sounds are the ones used by AI announcements, right? wouldn't that mean and AI that announces "phoron" would say "plasma"

Comment (tigercat2000) [inline code/defines/vox_sounds.dm]: hm.

**Comment by TheDZD** [comment]
@PJB3005, that holds true in other languages as well.

**Comment by monkeysfist101** [comment]
A big no from me. While it's not scientifically accurate, SS13 has never pretended to be scientifically accurate. Las a is one of the iconic parts of this game. As far as changing to match Bay, repeat after me, "Paradise is Paradise. We are not, nor will we ever be Bay. We are two different servers with two different things. We do not need to change for the sake of someone other than us."

**Comment by melandor0** [comment]
Don't change the color. But if you're going to change the name, for the love of god don't call it fucking phoron. Even unobtainium would be better than phoron, jesus christ.

**Comment by Fox-McCloud** [comment]
Ok guys. No name change here. This was all a big joke. @tigercat2000  wanted to practice a few things, but also wanted to pull a fast one on @fj45 because he knew he'd absssolutely hate this. Again, THIS IS WAS A JOKE. THERE IS NO ACTUAL NAME CHANGE.

Yes, coders and maintainers are mean =p

**Comment by tigercat2000** [comment]
we are

**Comment by PJB3005** [comment]
:sob:

**Comment by KasparoVy** [comment]
ban he!!

**Pull request closed without merging**